CRM/Upgrade/Incremental/sql/4.7.14.mysql.tpl

```diff
@@ -14,3 +14,9 @@ UPDATE civicrm_navigation SET
   `url` = 'civicrm/tag?reset=1'
 WHERE `name` = 'Tags (Categories)'
 AND `url` = 'civicrm/admin/tag?reset=1';
+
+--  CRM-19517 Disable all price fields nad price field options that use disabled fianncial types
+UPDATE civicrm_price_field_value cpfv
+INNER JOIN civicrm_financial_type cft oN cft.id = cpfv.financial_type_id
+SET cpfv.is_active = 0
+WHERE cft.is_active = 0;
```

Comment:
So it looks like this disables price field option values if its FT is disabled. Can you also disable price fields with no options if their FTs are disabled? Also, elsewhere in PR you handle case of null FTs (!); if you have seen evidence of them please handle setting is_active = false as appropriate using a left join to include those records as above.